n2y/blocks.py

```diff
@@ -80,6 +81,16 @@ def children_to_pandoc(self):
                         pandoc_ast.append(result)
         return pandoc_ast
 
+    def _retrieve_chidren(self):
```

Comment:
Since we added this, perhaps we could simplify the `__init__` method?

Reply:
I added that unnecessarily, I've just deleted it